### ISSUE TITLE
Structure for accessibility strings

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/number-pairs#150]
If there any strings in number-pairs-strings_en.json that are just for accessibility they should be moved under the `a11y` so that content is not translatable.

We are also transitioning to a structure that makes these keys more maintainable. Keys use the same value as a tandem name, and keys for `accessibleName` and `accessibleHelpText` make clear the intent. Something like this:

```json
  "a11y": {
    "sceneSelectionRadioButtonGroup": {
      "accessibleNamePattern": {
        "value": "Total Number {{value}}"
      },
      "accessibleHelpText": {
        "value": "Choose total for counting area."
      }
    }
  }
```

Sorry there isn't any written documentation for this yet. Let me know if you have any questions.
